test/test_rosbag/test/test_bag.py

```diff
@@ -188,6 +190,29 @@ def test_rosbag_filter(self):
 
         self.assertEquals(len(msgs), 5)
 
+    def test_trivial_rosbag_filter(self):
+        # regression test for issue #736
+
+        inbag_filename  = '/tmp/test_rosbag_filter__1.bag'
+        outbag1_filename = '/tmp/test_rosbag_filter__2.bag'
+        outbag2_filename = '/tmp/test_rosbag_filter__3.bag'
+
+        with rosbag.Bag(inbag_filename, 'w') as b:
+            for i in range(30):
+                msg = ColorRGBA()
+                t = genpy.Time.from_sec(i)
+                b.write('/ints' + str(i), msg, t)
+
+        expression = "True"
+        
+        # filtering multiple times should not affect the filtered rosbag
+        os.system('rosbag filter %s %s "%s"' % (inbag_filename, outbag1_filename, expression))
+        os.system('rosbag filter %s %s "%s"' % (outbag1_filename, outbag2_filename, expression))
+
+        outbag1_md5 = hashlib.md5(open(outbag1_filename,'r').read()).hexdigest()
+        outbag2_md5 = hashlib.md5(open(outbag2_filename,'r').read()).hexdigest()
```

Comment:
Please use `with` to scope each file read:

    with open(outbag1_filename, 'r') as h:
        outbag1_md5 = hashlib.md5(h.read()).hexdigest()

Also add a space after the comma before the `'r'` to comply with PEP8 (even if the existing code does a poor job with code style).